WIP: glusterd: massive cleanup

dmantipov opened the pull request:
This is not intended for an immediate review, and commit message
will be updated when/if this code passes the regression tests.

Signed-off-by: Dmitry Antipov <dantipov@cloudlinux.com>
Updates: #1000

gluster-ant commented:
CLANG-FORMAT FAILURE:
Before merging the patch, this diff needs to be considered for passing clang-format

```diff --git a/xlators/mgmt/glusterd/src/glusterd.c b/xlators/mgmt/glusterd/src/glusterd.c
index 53d671794..c00a62d81 100644
--- a/xlators/mgmt/glusterd/src/glusterd.c
+++ b/xlators/mgmt/glusterd/src/glusterd.c
@@ -1541,22 +1541,15 @@ init(xlator_t *this)
     struct {
         char *base;
         const char *name;
-    } *p, dirs[] = {
-        { workdir, "vols" },
-        { rundir, "vols" },
-        { workdir, "snaps" },
-        { workdir, "peers" },
-        { logdir, "bricks" },
+    } * p, dirs[] = {{workdir, "vols"},       {rundir, "vols"},
+                     {workdir, "snaps"},      {workdir, "peers"},
+                     {logdir, "bricks"},
 #ifdef BUILD_GNFS
-        { workdir, "nfs" },
+                     {workdir, "nfs"},
 #endif /* BUILD_GNFS */
-        { workdir, "bitd" },
-        { workdir, "scrub" },
-        { workdir, "glustershd" },
-        { workdir, "quotad" },
-        { workdir, "groups" },
-        { NULL, NULL }
-    };
+                     {workdir, "bitd"},       {workdir, "scrub"},
+                     {workdir, "glustershd"}, {workdir, "quotad"},
+                     {workdir, "groups"},     {NULL, NULL}};
 
 #if defined(RUN_WITH_MEMCHECK)
     vgtool = _gf_memcheck;
```

dmantipov commented:
/run regression

gluster-ant commented:
0 test(s) failed 


1 test(s) generated core 
./tests/000-flaky/bugs_core_multiplex-limit-issue-151.t

2 test(s) needed retry 
./tests/000-flaky/basic_mount-nfs-auth.t
./tests/000-flaky/bugs_core_multiplex-limit-issue-151.t

2 flaky test(s) marked as success even though they failed 
./tests/000-flaky/basic_mount-nfs-auth.t
./tests/000-flaky/bugs_core_multiplex-limit-issue-151.t
https://build.gluster.org/job/gh_centos7-regression/2527/

dmantipov commented:
/run regression

gluster-ant commented:
CLANG-FORMAT FAILURE:
Before merging the patch, this diff needs to be considered for passing clang-format

```diff --git a/xlators/mgmt/glusterd/src/glusterd.c b/xlators/mgmt/glusterd/src/glusterd.c
index 53d671794..c00a62d81 100644
--- a/xlators/mgmt/glusterd/src/glusterd.c
+++ b/xlators/mgmt/glusterd/src/glusterd.c
@@ -1541,22 +1541,15 @@ init(xlator_t *this)
     struct {
         char *base;
         const char *name;
-    } *p, dirs[] = {
-        { workdir, "vols" },
-        { rundir, "vols" },
-        { workdir, "snaps" },
-        { workdir, "peers" },
-        { logdir, "bricks" },
+    } * p, dirs[] = {{workdir, "vols"},       {rundir, "vols"},
+                     {workdir, "snaps"},      {workdir, "peers"},
+                     {logdir, "bricks"},
 #ifdef BUILD_GNFS
-        { workdir, "nfs" },
+                     {workdir, "nfs"},
 #endif /* BUILD_GNFS */
-        { workdir, "bitd" },
-        { workdir, "scrub" },
-        { workdir, "glustershd" },
-        { workdir, "quotad" },
-        { workdir, "groups" },
-        { NULL, NULL }
-    };
+                     {workdir, "bitd"},       {workdir, "scrub"},
+                     {workdir, "glustershd"}, {workdir, "quotad"},
+                     {workdir, "groups"},     {NULL, NULL}};
 
 #if defined(RUN_WITH_MEMCHECK)
     vgtool = _gf_memcheck;
```

gluster-ant commented:
1 test(s) failed 
./tests/basic/volume-snapshot-clone.t

1 test(s) generated core 
./tests/basic/volume-snapshot-clone.t

4 test(s) needed retry 
./tests/000-flaky/basic_mount-nfs-auth.t
./tests/000-flaky/bugs_nfs_bug-1116503.t
./tests/000-flaky/glusterd-restart-shd-mux.t
./tests/basic/volume-snapshot-clone.t

2 flaky test(s) marked as success even though they failed 
./tests/000-flaky/basic_mount-nfs-auth.t
./tests/000-flaky/bugs_nfs_bug-1116503.t
https://build.gluster.org/job/gh_centos7-regression/2528/

Commit:
glusterd: massive cleanup

This is not intended for an immediate review, and commit message
will be updated when/if this code passes the regression tests.

Signed-off-by: Dmitry Antipov <dantipov@cloudlinux.com>
Updates: #1000

dmantipov commented:
/run regression

gluster-ant commented:
CLANG-FORMAT FAILURE:
Before merging the patch, this diff needs to be considered for passing clang-format

```diff --git a/xlators/mgmt/glusterd/src/glusterd.c b/xlators/mgmt/glusterd/src/glusterd.c
index 53d671794..c00a62d81 100644
--- a/xlators/mgmt/glusterd/src/glusterd.c
+++ b/xlators/mgmt/glusterd/src/glusterd.c
@@ -1541,22 +1541,15 @@ init(xlator_t *this)
     struct {
         char *base;
         const char *name;
-    } *p, dirs[] = {
-        { workdir, "vols" },
-        { rundir, "vols" },
-        { workdir, "snaps" },
-        { workdir, "peers" },
-        { logdir, "bricks" },
+    } * p, dirs[] = {{workdir, "vols"},       {rundir, "vols"},
+                     {workdir, "snaps"},      {workdir, "peers"},
+                     {logdir, "bricks"},
 #ifdef BUILD_GNFS
-        { workdir, "nfs" },
+                     {workdir, "nfs"},
 #endif /* BUILD_GNFS */
-        { workdir, "bitd" },
-        { workdir, "scrub" },
-        { workdir, "glustershd" },
-        { workdir, "quotad" },
-        { workdir, "groups" },
-        { NULL, NULL }
-    };
+                     {workdir, "bitd"},       {workdir, "scrub"},
+                     {workdir, "glustershd"}, {workdir, "quotad"},
+                     {workdir, "groups"},     {NULL, NULL}};
 
 #if defined(RUN_WITH_MEMCHECK)
     vgtool = _gf_memcheck;
```

dmantipov commented:
Closed.
